Add a notice about removed Ui Elements into UPGRADE-2.0.md (#101)
monsieurbiz/SyliusRichEditorPlugin: UPGRADE-2.0.md

  Use the old Form here, it should work! Use the old `type` as an `alias` in the configuration
  to keep backward compatibility.  
  Use the old template as well for the frontend renderer. And write a new admin renderer the same way.

## Removed Product and Taxon elements

> We removed Product and Taxon elements because we want to keep a very simple plugin for the Rich Editor.  
> These elements will be available later in another plugin which will add many UI Elements for your store.  
> But actually you have to implements its manually in your project if you need.

Reviewer: ```suggestion
We removed Product and Taxon elements because we want to keep it simple for this plugin.  
These elements will be provided later in another plugin, which will be dedicated to fancy Ui Elements.  
Today you'll have to implement them manually on your project if you need them.
```
